ani.ter-markosyan/repo/Lesson_3/exercise_23/exercise_23.py

```diff
@@ -0,0 +1,4 @@
+#!usr/bin/evn python3
+number = int(input("Enter a five-digit number: "))
```

Comment:
LW: Check user input to avoid IndexError if the user input none five-digit number